hazelcast/src/main/java/com/hazelcast/map/impl/nearcache/NearCacheImpl.java

```diff
@@ -185,7 +203,7 @@ public void run() {
                         try {
                             lastCleanup = Clock.currentTimeMillis();
                             for (Map.Entry<Data, NearCacheRecord> entry : cache.entrySet()) {
-                                if (entry.getValue().isExpired(maxIdleMillis, timeToLiveMillis)) {
+                                if (entry.getValue().isExpired(getMaxIdleMillis(), getTimeToLiveMillis())) {
```

Comment:
The same goes for this code; additional fat getters the CPU needs to chew through.